packages/@aws-cdk/aws-ecs/lib/log-drivers/utils.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Remove undefined values from a dictionary
+ */
+export function removeEmpty<T>(x: { [key: string]: (T | undefined | string) }): { [key: string]: string } {
+    for (const key of Object.keys(x)) {
+      if (!x[key]) {
+        delete x[key];
+      }
+    }
+    return x as any;
+}
+
+/**
+ * Checks that a value is a positive integer
+ */
+export function ensurePositiveInteger(val: number) {
+    if (val && Number.isInteger(val) && val < 0) {
+        throw new Error(`\`${val}\` must be a positive integer.`);
+    }
+}
+
+/**
+ * Checks that a value is present in an array
+ */
+export function ensureInList(val: string, list: string[]) {
+    if (val && !(list.includes(val))) {
```

Comment:
Why the `if (val)` condition? Should be fine without, right?

Reply:
val was optional at one point and I was just moving too quick.

Reply:
updated.